packages/react/src/components/Code/index.tsx

```diff
@@ -64,17 +79,33 @@ const Code: React.ComponentType<React.PropsWithChildren<Props>> = ({
   }, [scrollable]);
 
   return (
-    <Highlighter
-      {...props}
-      PreTag={PreWithRef}
-      useInlineStyles={false}
-      className={classNames('Code', className, {
-        'Code--scrollable': scrollable
-      })}
-      tabIndex={scrollableRegion ? 0 : undefined}
-    >
-      {children}
-    </Highlighter>
+    <>
+      {(label || allowCopy) && (
+        <div className="CodeWrapper">
```

Comment:
This would better follow our [naming conventions](https://cauldron.dequelabs.com/style-guidelines#styles):

```suggestion
        <div className="Code__Header">
```